src/db/models/data_repo_commits/manager.rs

```diff
@@ -43,7 +43,8 @@ impl super::TxManager for DatabaseTransaction {
                     .push_bind(&dc.repo_type)
                     .push_bind(&dc.auth_commit_hash)
                     .push_bind(&dc.auth_commit_timestamp)
-                    .push_bind(&dc.publication_id);
+                    .push_bind(&dc.publication_id)
+                    .push_bind(dc.commit_type);
             });
```

Comment:
Last `dc` doesn't have `&` prefix (unlike previous version), just to confirm this is expected.